### PR TITLE
Alibi detect enhancements (Expose metrics with 0 value and include seldon_deployment_namespace tag to metric)

### PR DESCRIPTION
**Pull request by axsaucedo**
Fixes:
- [x] #2666
- [x] #2668

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Thu Nov 19 14:56:35 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=1

**Comment by seldondev** [comment]
Thu Nov 19 14:56:44 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=2

**Comment by seldondev** [comment]
Thu Nov 19 15:04:00 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=3

**Comment by seldondev** [comment]
Thu Nov 19 15:04:15 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=4

**Comment by seldondev** [comment]
Thu Nov 19 15:08:34 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=5

**Comment by seldondev** [comment]
Thu Nov 19 15:08:41 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=6

**Comment by seldondev** [comment]
Thu Nov 19 15:24:03 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=7

**Comment by seldondev** [comment]
Thu Nov 19 15:24:08 UTC 2020
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=8

**Comment by seldondev** [comment]
Thu Nov 19 16:04:42 UTC 2020
The logs for [lint] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=10

**Comment by seldondev** [comment]
Thu Nov 19 16:04:43 UTC 2020
The logs for [pr-build] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2672/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2672 --build=9

**Review by SachinVarghese — COMMENTED** [review]
/lgtm

File: components/alibi-detect-server/adserver/server.py [hunk 1]
```diff
@@ -27,6 +27,12 @@
 CESERVER_LOGLEVEL = os.environ.get("CESERVER_LOGLEVEL", "INFO").upper()
 logging.basicConfig(level=CESERVER_LOGLEVEL)
 
+DEFAULT_LABELS = {
+    "seldon_deployment_namespace": os.environ.get(
+        "SELDON_DEPLYOMENT_NAMESPACE", "NOT_IMPLEMENTED"
```
Comment (SachinVarghese) [inline components/alibi-detect-server/adserver/server.py:32]: Hey @axsaucedo missed a typo here

**Pull request merged**